src/Nethermind/Nethermind.Core/Crypto/Keccak.cs

```diff
@@ -151,6 +151,10 @@ public class Keccak : IEquatable<Keccak>, IComparable<Keccak>
         public Keccak(string hexString)
             : this(Extensions.Bytes.FromHexString(hexString)) { }
 
+        public Keccak(ReadOnlySpan<byte> bytes) : this(bytes.ToArray())
```

Comment:
This looks allocate'y. If this is the first step then maybe it's ok, but this might hide allocs in various places.

Reply:
Moved to places that uses it.